tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -112,3 +127,94 @@ def test_streams_resilient_to_broker_down(self):
         self.assert_produce_consume("after_broker_stop")
 
         self.kafka.stop()
+
+    def test_streams_runs_with_broker_down_initially(self):
+        self.kafka.start()
+        node = self.kafka.leader(self.inputTopic)
+        self.kafka.stop_node(node)
+
+        broker_down_initially_in_seconds = 60
+
+        configs = self.get_configs(extra_configs=",application.id=starting_wo_broker_id")
+
+        # start streams with broker down initially
+        processor = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor.start()
+
+        processor_2 = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor_2.start()
+
+        processor_3 = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor_3.start()
+
+        # wait for broker downtime beyond timeouts
+        time.sleep(broker_down_initially_in_seconds)
```

Comment:
Do we need this as we `wait_for_verification` in L155 ff anyway?

Reply:
ack